inference-engine/src/plugin_api/ie_ngraph_utils.hpp

```diff
@@ -126,5 +127,13 @@ inline Precision convertPrecision(const ::ngraph::element::Type& precision) {
     }
 }
 
+/**
+ * @brief Clones input network including all layers and internal data objects
+ * @note Blobs inside layers are reused
+ * @param network A network to clone
+ * @return A cloned object
+ */
+INFERENCE_ENGINE_API_CPP(std::shared_ptr<CNNNetwork>) cloneNetwork(const CNNNetwork& network);
```

Comment:
BTW, do we need to return shared_ptr? I think we can return `CNNNetwork` itself (which is a wrapper for shared_ptr)

Reply:
good point. Changed.